Correct the install path of man.local to fix following error:
    /yocto/build/tmp/sysroots/x86_64-linux/usr/share/groff/1.22.2/tmac/an-old.tmac:690: warning: can't find macro file `man.local'

Signed-off-by: Zhenhua Luo <zhenhua.luo@freescale.com>
Upstream-Status: Pending

diff --git a/tmac/Makefile.sub b/tmac/Makefile.sub
index 1506232..f1468c5 100644
--- a/tmac/Makefile.sub
+++ b/tmac/Makefile.sub
@@ -121,9 +121,9 @@ install_data: $(NORMALFILES) $(SPECIALFILES) man.local \
 	  $(RM) $(DESTDIR)$(mdocdir)/$$f; \
 	  $(INSTALL_DATA) $$f-s $(DESTDIR)$(mdocdir)/$$f; \
 	done
-	-test -f $(DESTDIR)$(localtmacdir)/man.local \
+	-test -f $(DESTDIR)$(tmacdir)/man.local \
 	  || $(INSTALL_DATA) $(srcdir)/man.local \
-	       $(DESTDIR)$(localtmacdir)/man.local
+	       $(DESTDIR)$(tmacdir)/man.local
 	-test -f $(DESTDIR)$(localtmacdir)/mdoc.local \
 	  || $(INSTALL_DATA) mdoc.local-s $(DESTDIR)$(localtmacdir)/mdoc.local
 
@@ -164,9 +164,9 @@ uninstall_sub:
 	$(RM) $(DESTDIR)$(tmacdir)/$(tmac_s_prefix)s.tmac
 	$(RM) $(DESTDIR)$(tmacdir)/$(tmac_an_prefix)an.tmac
 	$(RM) $(DESTDIR)$(tmacdir)/www.tmac
-	-if cmp -s $(DESTDIR)$(localtmacdir)/man.local \
+	-if cmp -s $(DESTDIR)$(tmacdir)/man.local \
 	           $(srcdir)/man.local; then \
-	  $(RM) $(DESTDIR)$(localtmacdir)/man.local; \
+	  $(RM) $(DESTDIR)$(tmacdir)/man.local; \
 	fi
 	-if cmp -s $(DESTDIR)$(localtmacdir)/mdoc.local \
 	           $(srcdir)/mdoc.local; then \
